welder.py

```diff
@@ -0,0 +1,50 @@
+import subprocess, os
+from flask import (Flask, request, abort, jsonify)
+
+app = Flask(__name__)  # create the application instance
+
+
+@app.route('/welder/api/syncTo', methods=['POST'])
+def sync_to():
+    if not request or 'destination' not in request.json:
+        abort(400)
```

Comment:
do you want a return statement after this? right now i think you'd return a 400 to the user and then continue executing this function

Reply:
I'll add the return for completeness but this is working as intended 🤷‍♂️

Reply:
i think i wanna see documentation saying that flask throws an exception and stops executing of this function in order for me to be convinced here